src/sqlancer/databend/DatabendSchema.java

```diff
@@ -339,6 +339,13 @@ private static List<DatabendColumn> getTableColumns(SQLConnection con, String ta
                     while (rs.next()) {
                         String columnName = rs.getString("column_name");
                         String dataType = rs.getString("data_type");
+                        if (dataType.contains("NULL")) {
+                            int idx = 0;
+                            while (idx < dataType.length() && dataType.charAt(idx) != ' ') {
```

Comment:
Perhaps `dataType.substring(0, dataType.indexOf(" "))` would be easier to understand?